types/tipset.go

```diff
@@ -53,6 +54,9 @@ func NewTipSet(blocks ...*Block) (TipSet, error) {
 			if blk.Height != height {
 				return UndefTipSet, errors.Errorf("Inconsistent block heights %d and %d", height, blk.Height)
 			}
+			if len(blk.Tickets) != numTickets {
+				return UndefTipSet, errors.Errorf("Inconsistent ticket array length %d and %d", len(blk.Tickets), numTickets)
```

Comment:
I think you might have what's being printed here flipped, should print the info about the block last, similar to the other errors around this.